client/a8c-for-agencies/components/pressable-usage-details/style.scss

```diff
@@ -86,4 +86,91 @@
 		@include heading-large;
 		margin-top: 5px;
 	}
+
+	/* Titan Email Addon styles */
+	.pressable-usage-details__addon-tag {
+		@include body-small;
+		background-color: var(--color-neutral-5);
+		color: var(--color-neutral-70);
+        font-size: 11px;
+		padding: 2px 6px;
+		border-radius: 4px;
+		margin-left: 6px;
+		text-transform: uppercase;
+	}
+
+	.pressable-usage-details__titan-domains {
+		display: flex;
+		flex-direction: column;
+		gap: 12px;
+		margin-top: 16px;
+        font-size: 14px;
+        font-weight: 600;
+	}
+
+	.pressable-usage-details__titan-domain-item {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		padding: 8px 0;
+		border-bottom: 1px solid var(--color-neutral-5);
+
+		&:last-child {
+			border-bottom: none;
+		}
+	}
+
+	.pressable-usage-details__titan-domain-info {
+		display: flex;
+		align-items: center;
+		flex-wrap: wrap;
+		gap: 8px;
+	}
+
+	.pressable-usage-details__titan-domain {
+		@include body-medium;
+		font-weight: 500;
```

Comment:
I'd probably avoid overriding this one and just use the standard core.. like `heading-medium`